roxctl/central/debug/download_diagnostics.go

```diff
@@ -69,6 +73,7 @@ To specify timeout, run  'roxctl' command:
 	c.PersistentFlags().StringVar(&outputDir, "output-dir", "", "output directory in which to store bundle")
 	c.PersistentFlags().StringSliceVar(&clusters, "clusters", nil, "comma separated list of sensor clusters from which logs should be collected")
 	c.PersistentFlags().StringVar(&since, "since", "", "timestamp starting when logs should be collected from sensor clusters")
+	c.PersistentFlags().BoolVar(&withAlerts, "with-alerts", false, "add alert data before/after reassessing policies")
```

Comment:
Not clear to me what does "before/after" mean. Before and/or after?

Reply:
I think "before and after" will be more clear. These are the files that will be output in the diagnostic bundle. Alerts before and after reassessing policies.